tasks/13_largest-of-four/largest-of-four.js

```diff
@@ -0,0 +1,10 @@
+function largestOfFour(array) {
+  return array.map((a)=> a.sort((x, y) => x-y).pop());
```

Comment:
Why do we need this implementation?

Reply:
Array.prototype.sort() compares elements after converting them to strings, so it doesn't work directly for comparing numbers - or maybe there's something I don't know?